src/migration.ts

```diff
@@ -6,7 +6,7 @@ import * as fs from 'fs/promises'
 import { exists } from './utils'
 
 interface LegacyState {
-    selectedManifest: { uri: vscode.Uri } | null
+    selectedManifest: { uri: { path: string } } | null
```

Comment:
PathLike maybe?

Reply:
Uri's API use string here, so it won't be a buffer

Reply:
Right by PathLike is defined to be `PathLike = string | buffer | URL`, it is more for readability purposes than anything else. string would have been fine everywhere we use PathLike

Reply:
But I guess it would be better in this case to match vscode's Uri's API

Reply:
Right